biz.aQute.remote/src/aQute/remote/agent/RedirectOutput.java

```diff
@@ -85,6 +69,31 @@ public void write(byte b[], int off, int len) {
 
 	public void flush() {
 		super.flush();
```

Comment:
I am not sure of the proper order here. Should the new code be executed before super.flush()?